configs/edge-config.yaml

```diff
@@ -35,6 +35,11 @@ detectors:
   # Blank id implies that no detectors have been configured to use motion detection
   # and/or local edge inference. The "default" templates are just examples of how one
   # might configure these features.
-  - detector_id: ''
-    motion_detection_template: "default"
+  - detector_id: 'det_2moISb6wdgxCgQPDu00KyFAM7GG'
```

Comment:
Are you intending to add real detector names here?

Reply:
Nope, thanks